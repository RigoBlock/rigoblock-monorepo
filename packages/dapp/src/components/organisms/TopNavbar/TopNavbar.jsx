import './TopNavbar.scss'
import * as ROUTES from '../../../constants/routes'
import Link, { LINK_SIZES } from '../../atoms/Link'
import React from 'react'
import UserMenu from '../../molecules/UserMenu'

const TopNavbar = () => (
  <div className="top-navbar">
    <Link to={ROUTES.DASHBOARD} size={LINK_SIZES.LARGE}>
      RigoBlock
    </Link>
    <UserMenu />
  </div>
)

export default TopNavbar
