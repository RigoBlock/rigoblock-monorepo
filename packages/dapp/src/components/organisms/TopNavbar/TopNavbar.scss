@import "../../_settings/variables.scss";

.top-navbar {
  height: $dist-xl;
  display: flex;
  padding: $dist-s $dist-s 0 $dist-m;
  align-items: center;
  justify-content: space-between;
}
