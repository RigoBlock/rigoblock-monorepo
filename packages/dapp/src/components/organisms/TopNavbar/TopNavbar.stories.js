import React from 'react'
import { storiesOf } from '@storybook/react'
import { MemoryRouter } from 'react-router-dom'
import TopNavbar from './TopNavbar'
import '../../_settings/_base.scss'

storiesOf('Organisms/TopNavbar', module)
  .addDecorator(story => <MemoryRouter>{story()}</MemoryRouter>)
  .add('default', () => <TopNavbar />)
