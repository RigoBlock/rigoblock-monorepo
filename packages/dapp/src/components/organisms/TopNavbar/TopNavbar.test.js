import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import TopNavbar from './TopNavbar.jsx'

describe('TopNavbar component', () => {
  it('renders correctly', () => {
    expect(toJson(shallow(<TopNavbar />))).toMatchSnapshot()
  })
})
