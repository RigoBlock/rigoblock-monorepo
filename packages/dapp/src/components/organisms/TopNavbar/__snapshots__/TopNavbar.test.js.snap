// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNavbar component renders correctly 1`] = `
<div
  className="top-navbar"
>
  <Link
    className="navbar-title"
    size="medium"
    to="/"
  >
    RigoBlock
  </Link>
  <UserMenu />
</div>
`;
